terasoluna-gfw-web/src/test/java/org/terasoluna/gfw/web/token/transaction/TransactionTokenIssue364InterceptorTest.java

```diff
@@ -0,0 +1,485 @@
+/*
```

Comment:
Since the URI is in the test class Setup () has been fixed, newly we create a test class.
The class name it will change.